Dockerfile

```diff
@@ -1,21 +1,41 @@
-FROM pmmp/pocketmine-mp:latest
+FROM ubuntu:18.04 as phpBuild
 
 USER root
-RUN apt-get update && apt-get install -y --no-install-recommends git
+
+RUN mkdir /php
+RUN apt-get update && apt-get install --no-install-recommends -y ca-certificates wget make autoconf automake libtool-bin m4 gzip bzip2 bison g++ git cmake pkg-config re2c
+WORKDIR /php
+RUN wget -q https://raw.githubusercontent.com/pmmp/php-build-scripts/stable/compile.sh
+RUN bash compile.sh -t linux64 -f -u -g -l -j
```

Comment:
These flags will need updating - `-u` is no longer used and `-l` now has a different purpose

also, `-j` expects a value for number of compile threads